foia_hub/templates/request/form.html

```diff
@@ -17,7 +17,8 @@
 
 <section class="request form">
 
-  <form class="request" method="get" action="#">
+  <form class="request" method="POST" action="/request/noop/">
+    <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}"/>
```

Comment:
I don't think you need the hidden field -- doesn't `{% csrf_token %}` do that for you?